common/src/main/java/com/gregtechceu/gtceu/integration/kjs/recipe/GTRecipeBuilderJS.java

```diff
@@ -98,7 +98,8 @@ public void deserialize() {
 
     @Override
     public void serialize() {
-        json.addProperty("recipe_type", backingBuilder.recipeType.registryName.toString());
+        backingBuilder.id(this.id);
+        //json.addProperty("type", backingBuilder.recipeType.registryName.toString());
```

Comment:
If this code is unused, it should be removed. Otherwise its implementation should be finished.